pkg/xds/envoy/listeners/v2/http_inbound_route_configurer_test.go

```diff
@@ -57,6 +57,7 @@ var _ = Describe("HttpInboundRouteConfigurer", func() {
             address:
               socketAddress:
                 address: 192.168.0.1
+                ipv4Compat: true
```

Comment:
Could you help me understand why are we using `ipv4compat`? do we actually use this compat mode anywhere? As far as I understand the docs, it enables a capability of using both IPV6 and V4 at the same time

Reply:
Not sure why, will just remove it and see if there are any obvious drawbacks